static/app/components/alert.tsx

```diff
@@ -165,21 +163,23 @@ const Alert = styled(
         className={classNames(type ? `ref-${type}` : '', className)}
         {...props}
       >
-        <MessageContainer>
-          {showIcon && <IconWrapper>{getIcon()}</IconWrapper>}
-          <StyledTextBlock>{children}</StyledTextBlock>
-          {(showExpand || defined(trailingItems)) && (
-            <TrailingItems>
-              {trailingItems}
-              {showExpand && <ExpandIcon isExpanded={isExpanded} />}
-            </TrailingItems>
+        {showIcon && <IconWrapper>{getIcon()}</IconWrapper>}
+        <ContentWrapper>
+          <MessageContainer>
+            <StyledTextBlock>{children}</StyledTextBlock>
+            {(showExpand || defined(trailingItems)) && (
+              <TrailingItems>
+                {trailingItems}
+                {showExpand && <ExpandIcon isExpanded={isExpanded} />}
+              </TrailingItems>
+            )}
+          </MessageContainer>
+          {showExpandItems && (
+            <ExpandContainer>
+              {Array.isArray(expand) ? expand.map(item => item) : expand}
+            </ExpandContainer>
           )}
-        </MessageContainer>
-        {showExpandItems && (
-          <ExpandContainer>
-            <DetailsContainer>{(expand || []).map(item => item)}</DetailsContainer>
-          </ExpandContainer>
```

Comment:
We should be able to pass single elements into `expand` in addition to just arrays.